ui/app/store/actions.js

```diff
@@ -1464,6 +1464,10 @@ export function setProviderType (type) {
     }
     dispatch(setPreviousProvider(currentProviderType))
     dispatch(updateProviderType(type))
+    // Tests should behave in same way as mainnet, but are using Localhost
+    if (!process.env.IN_TEST) {
+      dispatch(setFeatureFlag('advancedInlineGas', type !== 'mainnet'))
```

Comment:
Wouldn't this have the effect of _enabling_ advanced inline gas for all users when they switched to mainnet? We'd still want this to be disabled unless the user turns it on.

Reply:
:sweat_smile: @rekmarks this is covered by your comment